src/openzaak/components/documenten/admin.py

```diff
@@ -251,3 +273,22 @@ def get_object_actions(self, obj):
             link_to_related_objects(Gebruiksrechten, obj.canonical),
             link_to_related_objects(ObjectInformatieObject, obj.canonical),
         )
+
+    def get_urls(self):
```

Comment:
why are you not re-using the URL(s) from django-privates that are already provided?